### PR TITLE
Add some tests for StringInfo

### PR DESCRIPTION
**Pull request by hughbe**
- Brings line and branch coverage up to 100%
- Remove randomly generated data from the tests

### CODE REVIEW

**Comment by joshfree** [comment]
@ianhays PTAL

**Comment by ianhays** [comment]
Looks like most of the new InlineData strings cover the same scenarios that the randomly generated strings did (plus a few extra). Couldn't find any glaring exclusions.

Thanks again, Hugh! LGTM.

File: src/System.Globalization/tests/StringInfo/StringInfoCtor.cs [hunk 1]
```diff
@@ -9,30 +9,34 @@ namespace System.Globalization.Tests
...
         public void Ctor_Empty()
         {
             StringInfo stringInfo = new StringInfo();
             Assert.Equal(string.Empty, stringInfo.String);
+            Assert.Equal(0, stringInfo.LengthInTextElements);
         }
 
         public static IEnumerable<object[]> Ctor_String_TestData()
         {
-            yield return new object[] { s_randomDataGenerator.GetString(-55, false, MinStringLength, MaxStringLength) };
```
Comment (stephentoub) [inline src/System.Globalization/tests/StringInfo/StringInfoCtor.cs]: It looks like this was generating a 244 character string.  There doesn't appear to be anything as long in what you've replaced this with...?

Comment (hughbe) [inline src/System.Globalization/tests/StringInfo/StringInfoCtor.cs]: The ctor for StringInfo is pretty simple and doesn't depend on the string (just sets the inner `String` property), but I'll add this

Comment (hughbe) [inline src/System.Globalization/tests/StringInfo/StringInfoCtor.cs]: done

**Comment by stephentoub** [comment]
Test Innerloop OSX Debug Build and Test please
Test Innerloop OSX Release Build and Test please

**Pull request merged**